pkg/dbconn/metadatalock.go

```diff
@@ -66,7 +68,8 @@ func NewMetadataLock(ctx context.Context, dsn string, lockName string, logger lo
 	}
 
 	// Acquire the lock or return an error immediately
-	logger.Infof("attempting to acquire metadata lock: %s", lockName)
+	// We only Infof the initial acquisition.
+	logger.Infof("attempting to acquiring metadata lock: %s", lockName)
```

Comment:
grammar is weird.  Maybe just `acquiring metadata lock`?

Reply:
Sorry, my mistake. I will revert it.

(I was planning to change it, but then thought I reverted.)